packages/webpack/src/plugin.js

```diff
@@ -452,7 +463,9 @@ class LavaMoatPlugin {
              * @returns {plugin is import('webpack').WebpackPluginInstance}
              */
             (plugin) =>
-              !!plugin && typeof plugin === 'object' && plugin.constructor.name === 'HtmlWebpackPlugin'
+              !!plugin &&
+              typeof plugin === 'object' &&
+              plugin.constructor.name === 'HtmlWebpackPlugin'
           )
```

Comment:
```suggestion
              typeof plugin === 'object' &&
              plugin.constructor.name === 'HtmlWebpackPlugin'
          )
```
the typeof should imply truthiness?